types/light-sdk/tslint.json

```diff
@@ -0,0 +1,80 @@
+{ 
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Please don't disable the lint rules for new definitions, thanks! ❤️

Reply:
ok, i got it!